build/management/commands/build_rfaa_models.py

```diff
@@ -0,0 +1,1013 @@
+            except:
+                pass
+
+            ##### SIGNPROT
+            # beta_gamma = sd['beta_gamma']
+            # signprot = Protein.objects.get(entry_name=sd['signprot'])
+            # signprot_conf = ProteinConformation.objects.get(protein=signprot)
+            # if beta_gamma:
+            #     beta_protconf = ProteinConformation.objects.get(protein__entry_name='gbb1_human')
+            #     gamma_protconf = ProteinConformation.objects.get(protein__entry_name='gbg2_human')
+            #     sc = SignprotComplex.objects.get_or_create(alpha='B', protein=signprot, structure=struct,
+            #                                                beta_chain='C', gamma_chain='D', beta_protein=beta_protconf.protein, gamma_protein=gamma_protconf.protein)
+            # else:
+            #     sc = SignprotComplex.objects.get_or_create(alpha='B', protein=signprot, structure=struct,
+            #                                                beta_chain=None, gamma_chain=None, beta_protein=None, gamma_protein=None)
+            # struct.signprot_complex = sc[0]
+            # struct.save()
+
+            #### Adding metrics to StructureRFAAScores
+
+            try:
+                metrics = StructureRFAAScores.objects.get(structure=struct)
+            except StructureRFAAScores.DoesNotExist:
+                metrics = StructureRFAAScores()
+
+            metrics.structure = struct
+            metrics.pae_7tm = sd['pae_7tm']
+            metrics.plddt_mean = sd['plddt_mean']
+            metrics.save()
+
+            #Adding plDDT for rendering
+            resis = []
+            for chain in self.parsed_pdb:
+                for res in chain:
+                    if 'LG1' in res.id[0]:  # Residue name of Ligand in PDB
+                        continue
+                    plddt = res['C'].get_bfactor()
+                    try:
+                        if chain.get_id()=='A':
+                            res_obj = Residue.objects.get(protein_conformation__protein=con, sequence_number=res.get_id()[1])
+                        # elif chain.get_id()=='B':
+                        #     res_obj = Residue.objects.get(protein_conformation__protein=signprot, sequence_number=res.get_id()[1])
+                        # elif chain.get_id()=='C':
+                        #     res_obj = Residue.objects.get(protein_conformation__protein=beta_protconf.protein, sequence_number=res.get_id()[1])
+                        # elif chain.get_id()=='D':
+                        #     res_obj = Residue.objects.get(protein_conformation__protein=gamma_protconf.protein, sequence_number=res.get_id()[1])
+                        r = StructureModelpLDDT(structure=struct, residue=res_obj, pLDDT=plddt)
+                        resis.append(r)
+                    except Residue.DoesNotExist:
+                        continue
+            StructureModelpLDDT.objects.bulk_create(resis)
+
+            # try:
```

Comment:
please clean this part up as well, we'll add compute_interactions and contact network when we need it